client/src/TrustAndSafety.tsx

```diff
@@ -97,97 +60,103 @@ const useStyles = makeStyles((theme: Theme) => ({
-                    <Typography className={classes.header} variant="h3" component="h3" align="center">
-                        Our Community Guidelines
+            <Box className={`${classes.titleBox}`}>
+                <Container>
+                    <Typography variant="h3" component="h3" align="center">
+                        Trust, Safety, & Privacy
                     </Typography>
-                    <Typography className={classes.headerText} variant="body1" component="div">
+                    <Typography variant="body1" component="div">
                         {loremIpsum}
                     </Typography>
-                    <Box className={`${classes.row} ${classes.buttonRow}`}>
-                        <SmallDisplayCard
-                            headerText="Trust"
-                            bodyText={loremIpsum.slice(0,56)}
-                        />
-                        <SmallDisplayCard
-                            headerText="Safety"
-                            bodyText={loremIpsum.slice(0,56)}
-                        />
-                        <SmallDisplayCard
-                            headerText="Privacy"
-                            bodyText={loremIpsum.slice(0,56)}
-                        />
+                </Container>
+            </Box>
+            <Box className={`${classes.guidelinesContainer}`}>
+                <Container>
+                    <Box className={`${classes.guidelinesContent}`}>
+                        <Typography variant="h3" component="h3" align="center">
+                            Our Community Guidelines
+                        </Typography>
+                        <Typography variant="body1" component="div">
+                            {loremIpsum}
+                        </Typography>
+                        <Grid container direction="row" alignContent="space-between" spacing={2}>
+                            <Grid item xs={12} sm={4} container direction="column" alignItems="center">
+                                <SmallDisplayCard
+                                    headerText="Trust"
+                                    bodyText={loremIpsum.slice(0,56)}
+                                />
+                            </Grid>
+                            <Grid item xs={12} sm={4} container direction="column" alignItems="center">
+                                <SmallDisplayCard
+                                    headerText="Safety"
+                                    bodyText={loremIpsum.slice(0,56)}
+                                />
+                            </Grid>
+                            <Grid item xs={12} sm={4} container direction="column" alignItems="center">
+                                <SmallDisplayCard
+                                    headerText="Privacy"
+                                    bodyText={loremIpsum.slice(0,56)}
+                                />
+                            </Grid>
+                        </Grid>
```

Comment:
it's fine for this PR but it'd be good to get in the habit of keeping your cleanup out of a bigger PR like this
it's best to have each PR have a single responsibility

Reply:
Ok, got it